packages/office-ui-fabric-react/src/components/Dialog/examples/Dialog.Basic.Example.tsx

```diff
@@ -3,25 +3,36 @@ import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dia
 import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { getId } from 'office-ui-fabric-react/lib/Utilities';
 import { hiddenContentStyle, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
+import { ContextualMenu } from '../../ContextualMenu/index';
```

Comment:
remove the path based import from the example, replace with `office-ui-fabric-react/lib/ContextualMenu`